src/System.Text.Json/docs/writable_json_dom_spec.md

```diff
@@ -0,0 +1,240 @@
+# Writable JSON DOM for `System.Text.Json`
```

Comment:
nit: State what the acronym means fully the first time it shows up in the doc (at least for not commonly known ones). 
```suggestion
# Writable JSON Document Object Model (DOM) for `System.Text.Json`
```